src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -90,14 +90,6 @@ public void parse_compulsoryFieldMissing_failure() {
         assertParseFailure(parser, VALID_TITLE_BOB + PHONE_DESC_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB,
                 expectedMessage);
 
-        // missing phone prefix
```

Comment:
Shall discuss whether these fields should be optional in the team meeting later tonight